### PR TITLE
Payments edits

### PR DESCRIPTION
**Pull request by riccardo-ep**
adding support for Stripe and other payment gateways

### CODE REVIEW

File: src/datasources/CartsDataSource.ts [hunk 1]
```diff
@@ -16,15 +16,14 @@ export class CartsDataSource extends RESTDataSource {
   }
 
   async getCart(id) {
-    const { data } = await this.get(`/carts/${id}/`);
+    const results = await this.get(`/carts/${id}`, {
+      include: 'items'
+    });
+    var data = results.data;
+    data.included = results.included
     return data;
```
Comment (Danial-EP) [inline src/datasources/CartsDataSource.ts:24]:
```suggestion
    const { data, included }= await this.get(`/carts/${id}`, {
      include: 'items'
    });
    return { data, included }
```

File: src/datasources/OrdersDataSource.ts [hunk 2]
```diff
@@ -35,11 +35,14 @@ export class OrdersDataSource extends RESTDataSource {
     return data;
   }
 
-  async payForOrder(orderId, gateway, method) {
+  async payForOrder(orderId, gateway, method, payment, options) {
+    console.log(JSON.stringify(options));
```
Comment (Danial-EP) [inline src/datasources/OrdersDataSource.ts:39]:
```suggestion
  async payForOrder(orderId, gateway, method, payment, options) {
```

File: src/datasources/CartsDataSource.ts (hunk 1 above)
Comment (riccardo-ep) [reply src/datasources/CartsDataSource.ts:24]: I like it but that won't work because it returns 
"{"data":{"id":"....
instead of 
{"id":"

**Comment by Danial-EP** [comment]
@riccardo-ep since, I've done `yarn prettier-check` there was some conflict, I thought it is easier to fix them :), please double-check them.

**Review by Danial-EP — APPROVED** [review]

File: src/datasources/CartsDataSource.ts (hunk 1 above)
Comment (Danial-EP) [reply src/datasources/CartsDataSource.ts]: Sorry, my bad. How about this? Please feel free to revert my changes :) 
```
    const { data, included }= await this.get(`/carts/${id}`, {      include: 'items'    });    
    data.included = included
    return data
```

Comment (riccardo-ep) [reply src/datasources/CartsDataSource.ts]: perfect

**Comment by riccardo-ep** [comment]
closing this and opening a new one with the prettyfied code

**Pull request closed without merging**